libvips/foreign/exif.c

```diff
@@ -525,7 +525,9 @@ vips__exif_parse( VipsImage *image )
 		int orientation;
 
 		orientation = atoi( str );
-		orientation = VIPS_CLIP( 1, orientation, 8 );
+		if ( orientation < 0 || orientation > 8 ) {
```

Comment:
Should this be < 1? I don't think 0 is valid.

https://sirv.com/help/resources/rotate-photos-to-be-upright/#exif-values

Style nitpick: libvips style is no space before the `(`, and no `{}` for single line `if()`s. A ternary might be neater.